plugin/src/test/java/hudson/plugins/findbugs/workflow/WorkflowCompatibilityTest.java

```diff
@@ -0,0 +1,61 @@
+package hudson.plugins.findbugs.workflow;
+
+import java.io.File;
+
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import hudson.FilePath;
+
+import hudson.model.Result;
+
+import hudson.plugins.findbugs.FindBugsPublisher;
+
+/**
+ * Test workflow compatibility.
+ */
+public class WorkflowCompatibilityTest {
+
+    @Rule
```

Comment:
a classRule will wrap around the class rather than test method - so the setup cost is only incurred once.